sdk/redis/Azure.ResourceManager.Redis/src/ApiCompatBaseline.txt

```diff
@@ -1,4 +0,0 @@
-Compat issues with assembly Azure.ResourceManager.Redis:
```

Comment:
What is this file?

Reply:
It's a file that we can use it to suppress the api compat error. That why the previous PR with breaking changes could pass the CI.